### PR TITLE
Specify Set Window Size in more detail.

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -1648,10 +1647,17 @@
 
           <li><p>Return <a>success</a> with data <var>data</var>
         </ol>
+
+        <aside class="note">
+          <p>In some browsers the dimensions of the
```
Comment (andreastt) [inline webdriver-spec.html]: I would use UA here as “browsers” can be misleading.

File: webdriver-spec.html [hunk 2]
```diff
@@ -1665,16 +1671,62 @@
...
-          <dd>The "height" value refer to the javascript <code>window.outerheight</code> property. If the User Agent does not support this property, it represents the width of the whole OS window including window chrome and window resizing borders/handles.</li></dd>
 
+        <p>The Set Window Size command alters the size of the
+        operating system window corresponding to the <a>current
+        top-level browsing context</a>.</p>
+
+        <p>The <a>remote end steps</a> for <dfn>Set Window Size</dfn>
+        are:</p>
+        <ol>
+          <li><p>If the <a>current top level browsing context</a>
```
Comment (andreastt) [inline webdriver-spec.html]: top-level (with dash)

File: webdriver-spec.html [hunk 3]
```diff
@@ -1665,16 +1671,62 @@
...
+              is <a>no longer open</a>, return an error with
```
Comment (andreastt) [inline webdriver-spec.html]: `<a>error</a>`

File: webdriver-spec.html [hunk 4]
```diff
@@ -1665,16 +1671,62 @@
...
+              code <a>no such window</a>.</li>
+
+          <li><p>If the <a>remote end</a> does not support the
+               <a>Set Window Size</a> command for the <a>current top
+              level browsing context</a> for any reason, return an
```
Comment (andreastt) [inline webdriver-spec.html]: top-level

File: webdriver-spec.html [hunk 5]
```diff
@@ -1665,16 +1671,62 @@
...
+              error with code <a>unsupported operation</a>.</li>
```
Comment (andreastt) [inline webdriver-spec.html]: `<a>error</a>`

File: webdriver-spec.html [hunk 6]
```diff
@@ -1665,16 +1671,62 @@
...
+
+          <li><p>Let <var>width</var> be the result of
+              <a>getting a property</a> named <code>width</code>
```
Comment (andreastt) [inline webdriver-spec.html]: `"<code>width</code>"`

**Comment by AutomatedTester** [comment]
landed in 5710f65

**Pull request closed without merging**